src/main.rs

```diff
@@ -220,23 +217,35 @@ fn main() {
         }
     }
 
-    info!(
-        message = "Loading config.",
-        path = ?opts.config_path
-    );
-
-    let file = if let Some(file) = open_config(&opts.config_path) {
-        file
+    let mut config_paths = Vec::new();
+    for config_pattern in if !opts.config_paths.is_empty() {
+        opts.config_paths
     } else {
-        std::process::exit(exitcode::CONFIG);
-    };
+        DEFAULT_CONFIG_PATHS.to_vec()
+    } {
+        let matches: Vec<PathBuf> = glob(config_pattern.to_str().expect("no ability to glob"))
```

Comment:
These expect messages seem like they would be better to be error! logs? I would imagine we only want to panic on truly fatal errors.